Automattic/jetpack
# Release Script: Support multiple plugins

## Proposed changes:

This updates the release-plugin.sh script to add support for multiple plugins, and will automatically add the mu-wpcom-plugin if we're releasing jetpack.

### Other information:


## Jetpack product discussion


N/a
## Does this pull request change what data or activity we track or use?


no
## Testing instructions:






Hard to test so might be best to test next cycle. 

It should work by running `tools/release-plugin.sh jetpack 12.0-a.0` etc. Support for multiple plugins like `tools/release-plugin.sh jetpack 12.0-a.0 backup 1.5 search 1.5.0` etc. If no release version is specified, it will try to automatically determine the next versions.
